src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -17,11 +18,9 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public static class Base64UrlEncoder
```

Comment:
```suggestion
    [Obsolete("This class is obsolete and will be removed in a future release. Use System.Buffers.Text.Base64Url instead")]
    public static class Base64UrlEncoder
```

Should this class (and all methods) be marked as obsolete?

Reply:
That would be nice, but unfortunately there's some quirks about this class and the various methods that don't match exactly with the `Base64Url` API, so I'm not entirely sure that's feasible. I've seen quite a few usages of this downstream that probably wouldn't be able to migrate easily.